### PR TITLE
Sql input/output binding template updates

### PR DESCRIPTION
**Pull request by MaddyDev**
1. Update connection string attribute changes
2. Update the extension version

### CODE REVIEW

File: Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
           "ManualInstructions": [],
           "args": {
             "referenceType": "package",
-            "reference": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "1.0.84-preview",
+            "reference": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "1.0.112-preview",
```
Comment (chlafreniere) [inline Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json:50]: 112 or 122?

Comment (MaddyDev) [reply Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json:50]: fixed that, thanks!!

Comment (chlafreniere) [reply Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json:50]: still shows as 112 for input bindings here. is that intentional?

**Review by chlafreniere — APPROVED** [review]

File: Functions.Templates/Templates/SqlInputBinding-CSharp/SqlInputBindingCSharp.cs [hunk 2]
```diff
@@ -15,8 +15,8 @@ public static class SqlInputBindingCSharp
         public static IActionResult Run(
             [HttpTrigger(AuthorizationLevel.Function, "get", Route = null)] HttpRequest req,
             [Sql("SELECT * FROM object",
-            CommandType = System.Data.CommandType.Text,
-            ConnectionStringSetting = "SqlConnectionString")] IEnumerable<Object> result,
+            "SqlConnectionString",
+            CommandType = System.Data.CommandType.Text)] IEnumerable<Object> result,
```
Comment (lucyzhang929) [inline Functions.Templates/Templates/SqlInputBinding-CSharp/SqlInputBindingCSharp.cs:19]: The `CommandType` can be removed since we default to Text.

**Comment by MaddyDev** [comment]
@soninaren I've tested both input and output bindings templates and they work as expected. Please merge this in to be in sync with our latest extension version.

**Review by soninaren — APPROVED** [review]

**Pull request merged**